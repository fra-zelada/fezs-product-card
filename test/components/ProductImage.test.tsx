import React from 'react';
import renderer from 'react-test-renderer';
import ProductCard, { ProductImage } from '../../src/components';
import { product2 } from '../data/products';

describe('pruebas en ProductImage', () => {

    test('debe de hace match con snapshot', () => {

        const wrapper = renderer.create(
            <ProductImage img='imagen.jpg' />
        )
        
        expect(wrapper.toJSON()).toMatchSnapshot();

       }
      )

      test('debe de mostrar el componente con la imagen deltes producto', () => {

        const wrapper = renderer.create(
            <ProductCard product={product2} >

                {

                    () => (
                        <ProductImage />
                    )

                }

            </ProductCard>
        )

        expect(wrapper.toJSON()).toMatchSnapshot();
        
    })

    }
)



