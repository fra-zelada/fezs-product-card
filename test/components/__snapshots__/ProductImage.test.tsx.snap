// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pruebas en ProductImage debe de hace match con snapshot 1`] = `
<img
  alt="Product"
  className="productImg undefined "
  src="imagen.jpg"
/>
`;

exports[`pruebas en ProductImage debe de mostrar el componente con la imagen deltes producto 1`] = `
<div
  className=" productCard undefined"
>
  <img
    alt="Product"
    className="productImg undefined "
    src="./coffee-mug.png"
  />
</div>
`;
